react/features/blur/reducer.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import { ReducerRegistry } from '../base/redux';
+import { TOGGLE_BLUREFFECT } from './actionTypes';
+import { PersistenceRegistry } from '../base/storage';
+
+PersistenceRegistry.register('features/videoeffects', true, {
+    blurEnabled: false
+});
+
+ReducerRegistry.register('features/videoeffects', (state = {}, action) => {
```

Comment:
Usually the key for the reducer matches the name of the feature. Is there any reason to not use the same name in this case? If not please rename the feature to videoeffects or use 'features/blur' for the reducer key.